protocol-units/bridge/integration-tests/tests/client_l1move_l2move.rs

```diff
@@ -0,0 +1,455 @@
+use alloy::{
+	primitives::{address, keccak256},
+	providers::Provider,
+};
+use anyhow::Result;
+use aptos_sdk::types::account_address::AccountAddress;
+use aptos_sdk::coin_client::CoinClient;
+use aptos_types::PeerId;
+use bridge_config::Config;
+use bridge_integration_tests::utils as test_utils;
+use bridge_integration_tests::EthToMovementCallArgs;
+use bridge_integration_tests::HarnessMvtClient;
+use bridge_integration_tests::{TestHarness, TestHarnessFramework};
+use bridge_service::chains::{ethereum::types::EthAddress, movement::client_framework::MovementClientFramework};
+use bridge_service::chains::{
+	bridge_contracts::{BridgeContract, BridgeContractEvent}, ethereum::types::EthHash, movement::{event_monitoring::MovementMonitoring, utils::MovementHash}
+};
+use bridge_service::types::{
+	Amount, AssetType, BridgeAddress, BridgeTransferId, HashLock, HashLockPreImage,
+};
+use futures::StreamExt;
+use tokio::time::{sleep, Duration};
+use tokio::{self};
+use tracing::info;
+
+#[tokio::test]
+async fn test_movement_client_lock_transfer(
+) -> Result<(), anyhow::Error> {
+	let _ = tracing_subscriber::fmt().with_max_level(tracing::Level::DEBUG).try_init();
+
+	let _ = tracing_subscriber::fmt().with_max_level(tracing::Level::DEBUG).try_init();
```

Comment:
Dup line here, we only need to init `tracing_subscriber` once for a a given context.

Reply:
Removed